app/models/skin.rb

```diff
@@ -108,8 +108,13 @@ def valid_public_preview
     errors.add(:base, ts("You need to upload a screencap if you want to share your skin."))
   end
 
-  validates_presence_of :title
-  validates :title, uniqueness: { message: ts("must be unique"), case_sensitive: true }
+  validates :title, presence: true, uniqueness: { message: ts("must be unique"), case_sensitive: true }
```

Comment:
Since you're touching this, it would be great to i18n this by using the default locale for the error:

```suggestion
  validates :title, presence: true, uniqueness: { case_sensitive: true }
```
and [overriding it](https://guides.rubyonrails.org/i18n.html#error-message-scopes) in the `models/en.yml` file:

```
  activerecord:
    errors:
      models:
        skin:
          attributes:
            title:
              taken: must be unique
```